benches/helpers.rs

```diff
@@ -16,6 +18,7 @@ pub(crate) const ASYNC_METHODS: [&str; 3] = [SYNC_FAST_CALL, SYNC_MEM_CALL, SYNC
 
 // 1 KiB = 1024 bytes
 pub(crate) const KIB: usize = 1024;
+pub(crate) const SLOW_CALL: Duration = Duration::from_micros(50);
```

Comment:
changes the `slow_call` from 1ms to 50us

the bad thing is that we can't really compare old results when introducing this